Cargo.toml

```diff
@@ -16,20 +16,20 @@ license = "MIT OR Apache-2.0"
 default = []
 
 [dependencies]
-wgpu = "0.5"
-glsl-to-spirv = { version = "0.1", optional = true }
-log = "0.4"
-imgui = "0.4"
+wgpu = "~0.5.2"
```

Comment:
Why using tilde requirements versus the normal carrot? All these projects should follow cargo's version of semver, so the carrot version is actually the same as the carrot in this instance and once deps go to 1.0, minor versions should be able to be upgraded.

Additionally, please only specify the very minimum version needed. Unless you actually need `0.5.2`, you should always specify `0.5` so that in case some crate needs an older `=` version, downstream crates won't have dups.

Reply:
Welp makes sense. Also didn't know that the normal versions would default to cargo's SemVer. Thanks for the heads up, will correct it.